anchore_engine/services/policy_engine/engine/vulns/providers.py

```diff
@@ -914,6 +924,27 @@ def update_feed_group_counts(self) -> None:
         """
         DataFeeds.update_counts()
 
+    def is_image_vulnerabilities_updated(
+        self, image: Image, db_session, since: datetime.datetime
+    ):
+        """
+        Image vulnerabilities *may be* updated after feeds are synced. This function determines if the image vulnerabilities
+        have been updated by comparing the last_sync timestamp of the feed groups with the input timestamp
+
+        Copied over from EvaluationCacheManager._inputs_changed in synchronous_operations.py
+        """
+        # TODO: zhill - test more
+        feed_group_updated_list = [
+            group.last_sync
+            if group.last_sync is not None
+            else datetime.datetime.utcfromtimestamp(0)
+            for feed in get_all_feeds(db_session)
+            for group in feed.groups
+        ]
+        return (
```

Comment:
nit (feel free to ignore): my opinion here is that this is a little hard to read, could be improved with some code structure:
```
if feed_group_updated_list:
  return max(feed_group_updated_list) > since
else:
  return False
```
Furthermore, I feel the `since` variable brings into some question about the method naming. Instead of determining whether the image vulnerabilities _are_ updated, you're making sure they've been updated `since` a particular time. 

Perhaps the method could be renamed to something like: `are_image_vulns_updated_recently` or something more specific.

Reply:
Ack. This is the legacy code that was copied over. As for the naming, I was going for the s3 style get-if-modified-since format. I might go ahead with this merge but will give the naming more thought